docs/configuration/configuration.md

```diff
@@ -1020,37 +1020,58 @@ Serverset data must be in the JSON format, the Thrift format is not currently su
 scrape targets from [Container Monitor](https://github.com/joyent/rfd/blob/master/rfd/0027/README.md)
 discovery endpoints.
 
-The following meta labels are available on targets during relabeling:
+One of the following `<triton_role>` types can be configured to discover targets:
+
+#### `container`
+
+The `container` role discovers one target per "virtual machine" owned by the `account`. These are smartos zones or lx/kvm/bhyve branded zones.
+
+The following meta labels are available on targets during [relabeling](#relabel_config):
 
 * `__meta_triton_groups`: the list of groups belonging to the target joined by a comma separator
 * `__meta_triton_machine_alias`: the alias of the target container
 * `__meta_triton_machine_brand`: the brand of the target container
```

Comment:
Hmm, could this have a value of cn in the container role?

Generally if the cn role doesn't actually have a brand, we shouldn't force it.

Reply:
No, currently brand would be one of "joyent" (possibly with -suffix), "lx", "kvm" or "bhyve".
The intent was to expose that this is a compute node, if there is a better way to do this that would be fine too.

Reply:
That sounds like we're making up data then, if there's no brand coming from the cn mode then let's no add it.

Reply:
I've dropped the fixed-value brand from compute nodes.